data/channels.csv

```diff
@@ -1518,7 +1518,6 @@ AntofagastaTV.cl,Antofagasta TV,,,,CL,CL-AN,Antofagasta,c/CL,spa,general,FALSE,,
 ANTV.cv,ANTV,,,,CV,,Praia,c/CV,por,legislative,FALSE,,,,https://portais.parlamento.cv/canalwebtv/,https://i.imgur.com/f3UxmGY.png
 ANTV.id,ANTV,,,Intermedia Capital,ID,,,c/ID,ind,,FALSE,1993-03-01,,,https://www.antvklik.com/,https://upload.wikimedia.org/wikipedia/commons/thumb/2/2e/Antv_logo.svg/512px-Antv_logo.svg.png
 ANTV.pa,ANTV,,,,PA,,Panama City,c/PA,spa,legislative,FALSE,,,,https://www.asamblea.gob.pa/,https://i.imgur.com/UAqcj4Z.png
-ANTV.ve,ANTV,,,,VE,,,c/VE,spa,legislative,FALSE,2016-01-15,,,http://www.asambleanacional.gob.ve/,https://i.imgur.com/Qm88HT0.png

```

Comment:
Why did you decide to delete this channel?

Reply:
Beacause this exists. [AnNinhTV.vn](https://github.com/iptv-org/database/pull/8676/files/312373ca0abfb26bff516ba3ebaecdf7c0564327#diff-3091847231889fafb0ee438a9e920f7a181a899e198f95a4284fd9cb17f5f156R1472)
So I merge it into that one

Reply:
Ah my mistake. I was refering to an inexistant ANTV.vn.